typescript/schemas/src/internal/types.ts

```diff
@@ -1,6 +1,7 @@
 export type FoxglovePrimitive =
   | "string"
   | "float64"
+  | "int32"
   | "uint32"
   | "boolean"
   | "bytes"
```

Comment:
should we remove `time` and `duration` from this enum if they are now real schemas?

Reply:
It's not clear to me what "primitive" means since it looks like pretty much everything here represents time as a sec/nsec pair; maybe it's more of an indication to use "built-in" types?

We could remove it here and adjust all the code to generate identical schemas, but since this is a public export, I'm inclined to leave it alone for now.